### PR TITLE
Documentation Details

### PR DESCRIPTION
**Pull request by arivepr**
Simply adding documentation for adding templates with mock data.
Signaling in the readme that this is JUST for testing purposes as of this moment.

### CODE REVIEW

**Comment by arivepr** [comment]
Anything you guys can catch here is greatly appreciated

File: README.md [hunk 1]
```diff
@@ -15,3 +15,36 @@ making this approach much faster to make changes to the PDF.
...
 ```
+
+# Endpoints
+You currently have 2 choices for generating any of the available templates:
+
+The generate endpoint will automatically download the template if it is available (requires the above mentioned headers)
+* `POST http://localhost:8000/api/crc-pdf-generator/v1/generate` 
+
+The preview endpoint will instead return the pdf preview environment:
+* `GET http://localhost:8000/preview`
```
Comment (Hyperkid123) [inline README.md:26]: The preview is not accessible in the deployed environment. We should mention that something.

**Review by Hyperkid123 — COMMENTED** [review]
I can already tell we will have to change a few things with the data fetching but I don't have a full picture yet. That will change once we are on 3scale and can actually call the services. But this is good for now.

File: README.md (hunk 1 above)
Comment (arivepr) [reply README.md:26]: So just to clarify, that endpoint is exclusively for local development?

Comment (Hyperkid123) [reply README.md:26]: Yes. 3scale will expose only routes under: /api/crc-pdf-generator/v1/

And yes the plan is to expose this preview endpoint as well sometime in the future. And that future is mainly coming after we are on 3scale gateway.

**Review by Hyperkid123 — APPROVED** [review]

**Pull request merged**